### PR TITLE
Fix #5659 . Ignore accounting for jobs that are in X state.

### LINKED ISSUE #5659: Track job attempts individually in PyCondorPlugin
We find a problem in a tier0 instance: a job was not correctly removed by condor and it remained in status X for a long time. This mess up the accounting for the next retries. 
We are organizing jobs in the condor queue by WMAgentID, but in cases like this, we could have more than one job in condor per WMAgentID.
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/BossAir/Plugins/CondorPlugin.py#L1098
The WMAgent internal job tracking should only ever look at the job attempts that are active. If it kills a job attempt, that attempt has to drop out of tracking. Which means the tracking algorithm should ignore WMAgent job ids and only track glideinwms job ids. 
We should make sure we are protected against this problem in the PyCondorPlugin, I dont think it makes sense to fix the CondorPlugin as it is going to be deprecated soon.

### PR DESCRIPTION
**Pull request by tsarangi**
Fixed Exceptions

### CODE REVIEW

**Comment by ticoann** [comment]
@tsarangi, Tapas, can this be merged?

**Comment by tsarangi** [comment]
@ticoann 
I think so. Let's confirm with @lucacopa, if he has tested this !

**Comment by ticoann** [comment]
OK, I will wait for Luis' green light, Thanks,

**Comment by lucacopa** [comment]
Sorry, I didn't have time to work on this test. I will do it for sure next week

File: src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py [hunk 1]
```diff
@@ -1164,13 +1170,15 @@ def LogToScheddExitCodeMap(x):
                     cres=condor.read_events(logfileobj,1)
                     ulog=list(cres)
                     
-                    tmpDict["JobStatus"]=LogToScheddExitCodeMap(int(ulog[-1]["TriggerEventTypeNumber"]))
+                    _tmpStat = int(ulog[-1]["TriggerEventTypeNumber"])
```
Comment (lucacopa) [inline src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py]: @tsarangi this is conflicting with #5742

Comment (tsarangi) [inline src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py]: @lucacopa issue #5659 should be merge first and then #5672

**Comment by lucacopa** [comment]
@ticoann this is tested and working. You can merge it.

**Pull request merged**